trade-finance-manager-api/src/v1/controllers/deal.party-db.js

```diff
@@ -28,11 +28,28 @@ const getCompany = async (req, res) => {
   }
 };
 
-const getPartyUrn = async ({ companyRegNo }) => {
+const getPartyUrn = async ({ companyRegNo, companyName }) => {
```

Comment:
testing wise could we add some tests for this spying on what is called and making sure the right things are called for FF on and off please?